VirtoCommerce.PricingModule.Data/Services/PricingServiceImpl.cs

```diff
@@ -425,6 +426,27 @@ public virtual void DeletePricelistsAssignments(string[] ids)
                 ResetCache();
             }
         }
+
+        public virtual void DeletePricelistsAssignmentsByFilter(PricelistAssignmentsSearchCriteria criteria)
```

Comment:
I still think that having this method in IPricingSearchService is better. No static method needed.